osidb/middleware.py

```diff
@@ -32,3 +34,41 @@ def __call__(self, request):
             _logger.info(f"ACLs set from middleware for user {request.user.username}")
             set_user_acls(request.user.groups.all())
         return self.get_response(request)
+
+
+def OsidbHistoryMiddleware(get_response):
```

Comment:
I am thinking, we do we create a new middleware here which tampers with the `pghistory.middleware.HistoryMiddleware` and `pghistory.middleware.WSGIRequest`. Could we scrape the usage of the HistoryMiddleware and just use the `OsidbHistoryMiddleware`? Looking at the code, there is only the logic which adds user pk there, and we then replace it with the username or email, so why no just skip the pk step? :thinking:

Reply:
I had the same thought, it would be less awkward and complicated to ditch the bundled history middleware, I think